docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -32,6 +32,7 @@ The main highlights are:
   - [**Report time range**]({{< relref "#report-time-range" >}})
   - [**Organization-wide report settings**]({{< relref "#organization-wide-report-settings" >}})
   - [**Report grid layout**]({{< relref "#report-grid-layout" >}})
+- [**X-Ray data source plugin**]({{< relref "#x-ray-data-source-plugin" >}})
```

Comment:
Needs proper alignment.

Reply:
I think this is aligned properly as it is not an enterprise feature.